### PR TITLE
Feature/start end navigation

### PR DESCRIPTION
**Pull request by Escowei98**
### Description

This is the Pull Request Template for the Oeffis Project

## Checklist

- [x] My code follows the style guidelines
- [ ] I have documented my code
- [x] My changes generate no new warnings
- [x] I have linked the Issue
- [x] The Pull Request covers all the Checkpoints from the tagged Issue
- [x] I have noted the hours of work i have put in the issue

## I have edited following Parts of the project

- [x] Frontend
- [ ] Backend
- [ ] Github
- [ ] Root

## If tests are required did you make them?

- [x] No tests required
- [ ] Yes
- [ ] No

### CODE REVIEW

File: frontend/src/components/LiveNavigationInfo/LiveNavigationInfoComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+import { differenceInHours, differenceInMinutes, format } from "date-fns";
+import { chevronDownOutline, chevronUpOutline } from "ionicons/icons";
+import { useState } from "react";
+import "./LiveNavigationInfoComponent.css";
+interface LiveNavigationInfoComponentProps {
+  arrivalTime: Date;
+}
+
+const getHours = (arrivalTime: Date): string => {
+  // format returns duration + 1 hour thats why we subtract 60
```
Comment (LaraBiss) [inline frontend/src/components/LiveNavigationInfo/LiveNavigationInfoComponent.tsx:11]: comment duplicate

File: frontend/src/components/ResultRoutes/ResultRoutes.css [hunk 2]
```diff
@@ -6,7 +6,8 @@ ion-label {
     display: flex;
     flex-direction: column;
     align-items: center;
-    padding: 16px 16px 0px 16px;
+    padding: 16px 16px 16px 16px;
```
Comment (LaraBiss) [inline frontend/src/components/ResultRoutes/ResultRoutes.css:9]: padding: 16px reicht aus (optional: em wäre besser)

File: frontend/src/components/ResultRoutes/ResultRoutes.css [hunk 3]
```diff
@@ -41,3 +42,30 @@ IonButton {
 .swiper-div {
     z-index: -10;
 }
+
+.circle-button {
+    --border-radius: 50%;
```
Comment (LaraBiss) [inline frontend/src/components/ResultRoutes/ResultRoutes.css:47]: For what is --border-radius ? Maybe just border-radius

File: frontend/src/components/StepProgressComponent.tsx [hunk 4]
```diff
@@ -25,10 +25,14 @@ const StepProgressComponent: React.FC<StepProgressProps> = (props: StepProgressP
     </div>
     <div className="line-info">
       <IonLabel>
-        Linie: {props.step.line}
+        {
+          props.step.line && "Linie:" + props.step.line
```
Comment (LaraBiss) [inline frontend/src/components/StepProgressComponent.tsx:29]: unnecessary spacing

**Review by LaraBiss — CHANGES_REQUESTED** [review]

File: frontend/src/components/ResultRoutes/ResultRoutes.css (hunk 3 above)
Comment (Escowei98) [reply frontend/src/components/ResultRoutes/ResultRoutes.css:47]: for IonButton

**Review by LaraBiss — APPROVED** [review]

**Pull request merged**